packages/base-formula-engine/src/value-object/primitive-object.ts

```diff
@@ -204,7 +204,12 @@ export class NumberValueObject extends BaseValueObject {
         if (valueObject.isArray()) {
             return valueObject.plus(this);
         }
-        return this.plusBy(valueObject.getValue());
+        const object = this.plusBy(valueObject.getValue());
```

Comment:
@DR-Univer Please make sure this is appropriate.

Reply:
This part of code was provided by @DR-Univer 🤣